libs/core/testing/CMakeLists.txt

```diff
@@ -28,6 +29,6 @@ add_hpx_module(
   HEADERS ${testing_headers}
   COMPAT_HEADERS ${testing_compat_headers}
   MODULE_DEPENDENCIES hpx_assertion hpx_config hpx_format hpx_functional
-                      hpx_preprocessor hpx_util
+                      hpx_preprocessor hpx_util nanobench
```

Comment:
Will this work if nanobench is not fetched?

Reply:
I don't think so, this part is really incomplete, I tried fetching nanobench only for internal use by this module but while it builds, cmake throws some non fatal errors, as the nanobench module is not being properly exported. I am working on that